app.py

```diff
@@ -477,14 +478,15 @@ def get_latest_github_commits():
         # author (github name), date, and URL to the changes
         commit_name = commitsArr[repo_num].commit.message.split("\n")[0]
         commit_author = commitsArr[repo_num].commit.author.name
-        commit_date = str(commitsArr[repo_num].commit.author.date)
+        # raw date input from pygithub
+        raw_date = str(commitsArr[repo_num].commit.author.date)
```

Comment:
Do you want to use `.isoformat()` here?

Reply:
it defaults to iso format

Reply:
like i mean pygithub just returns in iso

Reply:
Ill add it anyways though

Reply:
the string for raw_date also isn't a datetime object

Reply:
Got it, I assumed for some reason it was a date time